src/pages/Register.js

```diff
@@ -13,14 +14,27 @@ export class Register extends React.Component {
     }
 
     onClick = () => {
-        if (this.validateName() &&
+        if (!this.validateForm()) return;
+
+        register({
+            'name': this.state.name,
+            'email': this.state.email,
+            'password': this.state.password
+        })
+            .then((result) => {
+                login(result);
+                alert("Registration successful!")
```

Comment:
I think the user after login should be redirected to dashboard (or any other screen) and this alert is redundand.

I suggest avoiding using alerts if it's possible.